akka-stream-tests/src/test/scala/akka/stream/impl/SynchronousIterablePublisherSpec.scala

```diff
@@ -1,172 +0,0 @@
-/**
- * Copyright (C) 2014 Typesafe Inc. <http://www.typesafe.com>
- */
-package akka.stream.impl
-
-import scala.collection.immutable
-import scala.concurrent.duration._
-import akka.stream.testkit.AkkaSpec
-import akka.stream.testkit.StreamTestKit
-import akka.testkit.TestProbe
-import org.reactivestreams.{ Subscriber, Subscription }
-
-class SynchronousIterablePublisherSpec extends AkkaSpec {
-
-  "A SynchronousPublisherFromIterable" must {
```

Comment:
Wait. We still have SynchronousIterablePublisher right? I only removed the Iterable/IteratorPublisher stuff, but this one still exists. For example Source.single uses SynchronousIterablePublisher and it might be the fastest actually, much faster than the Stage based Iterable Source.